Cargo.toml

```diff
@@ -64,7 +65,7 @@ members = [
     "trogging",
     "wal_inspect",
     "wal",
-    "workspace-hack",
```

Comment:
I think the client is in here twice now by accident, I think we can remove this one.

Reply:
Indeed, I can clean this up. Likely did something out of order with `cargo hakari`.